contracts/remote-address-validator/RemoteAddressValidator.sol

```diff
@@ -15,7 +15,7 @@ contract RemoteAddressValidator is IRemoteAddressValidator, Upgradable {
 
     mapping(string => bytes32) public remoteAddressHashes;
     mapping(string => string) public remoteAddresses;
-    mapping(string => bool) public supportedByGateway;
+    string public chainName;
```

Comment:
this might cause issues with upgradability later. shall we add eternal storage in a future PR?